README.md

```diff
@@ -7,46 +7,17 @@ It is supposed to:
 + Talk to TiKV for reading/writing data and the resulted data is encoded/decoded just like what we do in TiDB.
```

Comment:
We may add a section named Overview here.

Reply:
these statements are the product capability definition of the java client. I think we can optimize it in the future once we have conducted a better one.